### ISSUE TITLE
first paragraph is missing the drop cap when feature image is small

### ISSUE THREAD

**Issue by alexgreen** [TheTyee/design-article.thetyee.ca#202]

**Comment by MrBryan** [comment]
url pls

**Comment by phillipadsmith** [comment]
all tla now wtf  ;-)

**Comment by MrBryan** [comment]
haha

**Comment by MrBryan** [comment]
i mean, hhh

**Comment by MrBryan** [comment]
You're probably talking about this story  http://preview.thetyee.ca/News/2016/06/22/BC-Ban-Big-Bucks/  The drop cap is missing because I set a rule that the paragraph has to be a certain length in characters (120 currently) or else it looks funny with dropcap.  The need is less obvious in these ones because the text column is narrower which would let it accomodate the tall character with adequate text around it but I don't think we need to complicate the rule that much (detect smaller image and don't withhold the drop-cap)

**Comment by alexgreen** [comment]
(Yes, that's the page! sorry for url omission.)

This is case where it would be great to extend the existing class rather than eliminate it.
I.e: 
`<p class="lead-in lead-in--short-para" data-dev-object-descrip="molecules/text/lead-in-text" data-dev-status="IN-PROGRESS">`

By default lead-in--short-para would be styled like this in article.css :

```
.lead-in--short-para:first-letter {
    display: inherit;
    overflow: inherit;
    float: inherit;
    padding-right: inherit;
    padding-bottom:inherit;
    margin-top: inherit;
    margin-bottom: inherit;
    font-family: inherit;
    font-weight: inherit;
    font-size: inherit;
    line-height: inherit;
    text-transform: inherit;
    color: inherit;
}
```

This might sound convoluted, but it allows me to easily target the element for alternate styling rather than eliminating the possibility of styling it.
Is that possible?

**Comment by alexgreen** [comment]
Actually this would be the proper way to do it:

```
.lead-in--short_para:first-letter {
    all: inherit;
}
```

... but guess what? No version IE supports the "all" property. So, I would go with

```
.lead-in--short-para:first-letter {
    all: inherit;
    /* fallbacks required (see http://caniuse.com/#feat=css-all ): */
    display: inherit;
    overflow: inherit;
    float: inherit;
    padding-right: inherit;
    padding-bottom:inherit;
    margin-top: inherit;
    margin-bottom: inherit;
    font-family: inherit;
    font-weight: inherit;
    font-size: inherit;
    line-height: inherit;
    text-transform: inherit;
    color: inherit;
}

```

**Comment by MrBryan** [comment]
Yes it's easy for me to detect the length of the paragraph with bric so I can apply various classes / whatever. 

Or we could even add a hook into load and screen resize event that detects the height of the paragraph as well which is probably more key

**Comment by alexgreen** [comment]
Cool. I think whatever is simplest and creates the least stress for the browser is probably the way to go.

**Comment by alexgreen** [comment]
Ok, please write the javascript to apply `class="lead-in"` when the height of the `<p>` element is equal to or greater than 75px. If the element is less than 75px it should apply `class="lead-in lead-in--short-para"`

**Comment by MrBryan** [comment]
ok will say look at this post-launch unless we wrap everything else up

**Comment by alexgreen** [comment]
That sounds wise, not a big priority.

**Comment by phillipadsmith** [comment]
Another drop cap-related issue:
<img width="628" alt="un_tells_canada_to_clean_up_its_act_on_inequality__social_rights___the_tyee" src="https://cloud.githubusercontent.com/assets/34128/16674095/a69a8080-4469-11e6-9f3c-284910aee1a2.png">
http://preview.thetyee.ca/News/2016/03/08/UN-Tells-Canada-Clean-Up-Inequality/

**Comment by MrBryan** [comment]
I set the drop cap to min 250 characters up from 120 to handle these cases for now. Repreviewed the story, no drop cap now.

**Comment by phillipadsmith** [comment]
> I set the drop cap to min 250 characters up from 120 to handle these cases for now. Repreviewed the story, no drop cap now.

👍
